lib/samples/create_and_edit_geometries/create_and_edit_geometries.dart

```diff
@@ -399,7 +393,7 @@ class _CreateAndEditGeometriesState extends State<CreateAndEditGeometries>
                   Tooltip(
                     message: 'Undo',
                     child: ElevatedButton(
-                      style: _buttonStyle,
+                      style: Theme.of(context).elevatedButtonTheme.style,
```

Comment:
Is this redundant? Won't an ElevatedButton automatically pick up the ThemeData's elevatedButtonTheme? I think we can remove the `style:` property of all the `ElevatedButton`s and they'll automatically inherit the proper theme.